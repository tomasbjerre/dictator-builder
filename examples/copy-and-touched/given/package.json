{
  "name": "my-app",
  "version": "1.2.3",
  "description": "......."
}