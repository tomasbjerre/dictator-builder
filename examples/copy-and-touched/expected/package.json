{
  "name": "my-app",
  "version": "1.2.3",
  "description": ".......",
  "repository": {
    "type": "git",
    "url": "git://github.com/react-boilerplate/react-boilerplate.git"
  },
  "engines": {
    "npm": ">=5",
    "node": ">=8.15.1"
  },
  "author": "Max Stoiber",
  "license": "MIT",
  "scripts": {
    "test:watch": "cross-env NODE_ENV=test jest --watchAll",
    "coveralls": "cat ./coverage/lcov.info | coveralls",
    "prettify": "prettier --write"
  },
  "browserslist": [
    "last 2 versions",
    "> 1%",
    "IE 10"
  ],
  "lint-staged": {
    "*.js": [
      "npm run lint:eslint:fix",
      "git add --force"
    ],
    "*.json": [
      "prettier --write",
      "git add --force"
    ]
  },
  "pre-commit": "lint:staged",
  "resolutions": {
    "babel-core": "7.0.0-bridge.0"
  },
  "dependencies": {
    "@babel/polyfill": "7.4.3",
    "chalk": "2.4.2"
  },
  "devDependencies": {
    "@babel/cli": "7.4.3",
    "@babel/core": "7.4.3"
  }
}